orchestrator/careplancontributor/webhook/eventhandler_test.go

```diff
@@ -0,0 +1,102 @@
+package webhook
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	events "github.com/SanteonNL/orca/orchestrator/careplancontributor/event"
+	"github.com/stretchr/testify/require"
+)
+
+type unmarshallable struct{}
+
+func (u unmarshallable) MarshalJSON() ([]byte, error) {
+	return nil, errors.New("fail")
```

Comment:
I don't really care about these test cases (Marshal only fals if it's passed a function pointer), but it doesn't hurt much.

Reply:
I tend to test all branches but it might be a bit much